libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,164 @@
+            if (time.isValid()) {
+                date.setHours(time.hours());
+                date.setMinutes(time.minutes());
+            }
+
+            onChange(date);
+        };
+
+        return (
+            <div className={cx(className, isTimeForStaticPeriodEnabled && "gd-flex-row")}>
+                {isMobile ? (
+                    <DateRangePickerInputFieldBody
+                        type="date"
+                        className={cx(
+                            "s-date-range-picker-date",
+                            "gd-date-range-picker-input",
+                            "gd-date-range-picker-input-native",
+                            error && "gd-date-range-picker-input-error",
+                        )}
+                        placeholder={placeholderDate}
+                        onChange={(event) =>
+                            onDateChange(convertPlatformDateStringToDate(event.target.value))
+                        }
+                        value={getFormattedDate()}
+                    />
+                ) : (
+                    <DateRangePickerInputField
+                        className={cx(
+                            `s-date-range-picker-date`,
+                            error && "gd-date-range-picker-input-error",
+                        )}
+                        classNameCalendar={`s-date-range-calendar`}
+                        ref={ref}
+                        onDayChange={onDateChange}
+                        value={value}
+                        format={dateFormat}
+                        placeholder={placeholderDate}
+                        dayPickerProps={{
+                            ...dayPickerPropsWithDefaults,
+                            onDayClick: handleDayClick,
+                        }}
+                        // showOverlay={placeholderDate ==="to"} // Always shows the calendar, useful for CSS debugging
+                    />
+                )}
+                {isTimeForStaticPeriodEnabled && (
+                    <span
+                        className={cx(
+                            "gd-date-range-picker-input",
+                            "gd-date-range-picker-input-time",
+                            "s-date-range-picker-input-time",
+                        )}
+                    >
+                        <span className="gd-icon-clock" />
+                        <input
+                            type="time"
+                            className="input-text"
+                            onChange={(event) => {
+                                onTimeChange(event.target.value);
+                            }}
+                            value={getFormattedTime()}
+                        />
+                    </span>
+                )}
+            </div>
+        );
+    },
+);
+
+export const DateTimePickerWithInt = injectIntl(DateTimePickerComponent, { forwardRef: true });
+
+// eslint-disable-next-line react/display-name
```

Comment:
Is this necessary in the both cases in this file? Can the display name be set to satisfy the rule?
https://stackoverflow.com/questions/67992894/component-definition-is-missing-display-name-for-forwardref

Reply:
fixed